### ISSUE TITLE
Python 3.9 required for Typing 

### ISSUE THREAD

**Issue by artnod78** [Julian-O/PySLOBS#2]
When I try to import SlobsConnection, I have an error with AudioService.

```Traceback (most recent call last):
  File "myscript.py", line 4, in <module>
    from pyslobs import SlobsConnection
  File "D:\Apps\Python\Python38-32\lib\site-packages\pyslobs\__init__.py", line 3, in <module>
    from .slobs.audioservice import AudioService
  File "D:\Apps\Python\Python38-32\lib\site-packages\pyslobs\slobs\audioservice.py", line 8, in <module>
    class AudioService(SlobsService):
  File "D:\Apps\Python\Python38-32\lib\site-packages\pyslobs\slobs\audioservice.py", line 18, in AudioService
    async def get_sources(self) -> list[AudioSource]:
TypeError: 'type' object is not subscriptable
```

Else, thanks for work.  :)

**Comment by artnod78** [comment]
I have updated python to 3.9 and it works fine.

**Comment by Julian-O** [comment]
Interesting. 

This code depends on PEP-585, which was only implemented in Python 3.9.

My understanding was it would be ignored by the Python interpreter in Python 3.8, and thus be no problem, but that understanding was clearly wrong.

I need to either stop using this form of type-hinting, or explicitly specify that Python 3.9 is the minimum version acceptable.

Thanks for the feedback.

**Comment by Julian-O** [comment]
I have taken the lazy way out, and specified that Python 3.9 is required.

However, if there are users who have a good reason to need Python 3.8 or 3.7 support, please raise an issue and I will look into it.

**Comment by mrtylmz555** [comment]
When ı run this code, output is "Waiting for luamacros script...".

Can you help me?

**Comment by Julian-O** [comment]
@mrtylmz555: If you have an issue unrelated to Python 3.9 being required for, please create a new issue in GitHub.

(Short version: This code does not use lua or luamacros, and is not the cause of that output. You might have installed some software that uses luamacros to rebind hotkeys on a second keyboard and link them to directly to StreamLabs OBS, or to a Python script that uses this package to achieve an effect on StreamLabs OBS, but I can't imagine a scenario where my code would output that. If you do create an issue, please give a *lot* more context about the software you are using and where the output appeared.)
